@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAttitudeHold.sqf

```diff
@@ -0,0 +1,48 @@
+params ["_heli", "_deltaTime"];
+
+//Roll
+private _pidRoll  = _heli getVariable "fza_sfmplus_pid_roll";
+//Pitch
+private _pidPitch = _heli getVariable "fza_sfmplus_pid_pitch";
+
+//
+private _subMode  = _heli getVariable "fza_ah64_attHoldSubMode";
+private _curVelX  = (velocityModelSpace _heli # 0) * -1.0;
+private _curVelY  = velocityModelSpace _heli # 1;
```

Comment:
Could you maybe call `velocityModelSpace` once?

Reply:
Fixed